src/altinn-app-frontend/src/components/summary/SummaryBoilerplate.tsx

```diff
@@ -13,6 +14,7 @@ export interface SummaryBoilerplateProps {
   changeText: string;
   label: any;
   readOnlyComponent?: boolean;
+  display?: SummaryDisplayProperties;
```

Comment:
This should probably be added to `ILayoutCompSummary` instead, and this should extend from that interface. That would give you intellisense on `component.display...` after checking `if (component.type === 'Summary')` :smiley:

Reply:
Ah, great tip! Was able to get rid of several props that were already defined in `ILayoutCompSummary`. Will update jsonschema and make sure that all props are defined there.